README.md

```diff
@@ -520,6 +521,17 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Terraform 
+
+- [Terraform Study Guide](https://github.com/ari-hacks/terraform-study-guide)
+- [Zeal Vora - Udemy course](https://www.udemy.com/course/terraform-beginner-to-advanced/)
```

Comment:
Replace with the title of the course

Reply:
Thanks @shahednasser, I have updated with title of the course